corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -403,6 +402,12 @@ def get_properties_by_case_type(self):
         for case_properties in case_properties_by_case_type.values():
             case_properties.update(self.defaults)
 
+        for case_type, case_properties in case_properties_by_case_type.items():
+            for prop in case_properties:
+                if prop == 'owner_id':
+                    case_properties_by_case_type[case_type].remove(prop)
+                    case_properties_by_case_type[case_type].add('@owner_id')
```

Comment:
I thought it was not safe to mutate a set while iterating over it like this, so I'm surprised this code even works at all. Do the tests ever use more than a single-element set? Suggested change:
```suggestion
            for prop in list(case_properties):
                if prop == 'owner_id':
                    case_properties.remove(prop)
                    case_properties.add('@owner_id')
```

Reply:
very good catch